### PR TITLE
Add engine-install-url flag for machine

### PR DESCRIPTION
**Pull request by cjellick**

### CODE REVIEW

File: code/implementation/docker/machine/src/main/resources/schema/project/project-auth.json.d/machine.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
         "machine.authKey" : "cr",
         "machine.extractedConfig" : "",
         "machine.labels": "cr",
+        "machine.engineInstallUrl": "r",
```
Comment (wlan0) [inline code/implementation/docker/machine/src/main/resources/schema/project/project-auth.json.d/machine.json]: Should this be "cr"?

File: tests/integration/cattletest/core/test_authorization.py [hunk 2]
```diff
@@ -1379,6 +1381,7 @@ def test_machine(admin_user_client, user_client, service_client,
         'authCertificateAuthority': 'cr',
         'labels': 'cr',
         'authKey': 'cr',
+        'engineInstallUrl': 'r',
```
Comment (wlan0) [inline tests/integration/cattletest/core/test_authorization.py]: If projects get create access too, then this should be changed to "cr"

File: tests/integration/cattletest/core/test_authorization.py [hunk 3]
```diff
@@ -1408,6 +1411,7 @@ def test_machine(admin_user_client, user_client, service_client,
...
```
Comment (wlan0) [inline tests/integration/cattletest/core/test_authorization.py]: same as above

File: code/implementation/docker/machine/src/main/resources/schema/base/machine.json.d/machine.json [hunk 4]
```diff
@@ -26,6 +26,10 @@
             "type": "map[string]",
             "nullable": true
         },
+        "engineInstallUrl": {
+            "type": "string",
+            "nullable": true
```
Comment (alena1108) [inline code/implementation/docker/machine/src/main/resources/schema/base/machine.json.d/machine.json]: :) I always forget to put nullable:true

**Comment by alena1108** [comment]
LGTM

**Comment by wlan0** [comment]
LGTM

**Pull request merged**